invenio_app_rdm/records_ui/views/filters.py

```diff
@@ -183,9 +184,10 @@ def custom_fields_search(field, field_value, field_cfg=None):
         if not locale:
             locale = current_app.config.get("BABEL_DEFAULT_LOCALE", "en")
         # example: cern:experiments.title.en
-        namespace_string = "\:".join(namespace_array) + f".{localised_title}.{locale}"
+        namespace_string = ":".join(namespace_array) + f".{localised_title}.{locale}"
```

Comment:
this change will break the searches in custom fields
basically if you search for a specific field
`q=custom_fields.cern\:experiments:ATLAS`
the slash prevents the custom field namespace separator `:` to be interpreted as a part of lucene syntax

Reply:
could you add a comment above, discouraging people from removing this slash ?:)

Reply:
this has been moved to a separate PR